It works now.

```
[14:12:45] [Test thread/ERROR] (fabric-client-gametest-api-v1) Client gametests failed with an exception
 java.lang.RuntimeException: test
        at knot/net.fabricmc.fabric.test.client.gametest.ClientGameTestTest.lambda$runTest$0(ClientGameTestTest.java:85) ~[testmodClient/:?]
        at knot/net.fabricmc.fabric.impl.client.gametest.ThreadingImpl.runTaskOnThisThread(ThreadingImpl.java:203) ~[fabric-client-gametest-api-v1-1.0.0+db066d5704-dev.jar:?]
        at Async Stack Trace..(Unknown Source) ~[?:?]
        at knot/net.fabricmc.fabric.impl.client.gametest.ThreadingImpl.runTaskOnOtherThread(ThreadingImpl.java:195) ~[fabric-client-gametest-api-v1-1.0.0+db066d5704-dev.jar:?]
        at knot/net.fabricmc.fabric.impl.client.gametest.ThreadingImpl.runOnServer(ThreadingImpl.java:179) ~[fabric-client-gametest-api-v1-1.0.0+db066d5704-dev.jar:?]
        at knot/net.fabricmc.fabric.test.client.gametest.ClientGameTestTest.runTest(ClientGameTestTest.java:84) ~[testmodClient/:?]
        at knot/net.fabricmc.fabric.impl.client.gametest.FabricClientGameTestRunner.lambda$start$0(FabricClientGameTestRunner.java:43) ~[fabric-client-gametest-api-v1-1.0.0+db066d5704-dev.jar:?]
        at knot/net.fabricmc.fabric.impl.client.gametest.ThreadingImpl.lambda$runTestThread$1(ThreadingImpl.java:129) ~[fabric-client-gametest-api-v1-1.0.0+db066d5704-dev.jar:?]
        at java.base/java.lang.Thread.run(Thread.java:1583) [?:?]
[14:12:45] [Server thread/INFO] (ServerChunkLifecycleTests) Loaded 3 freshly generated chunks in minecraft:overworld during tick #223
[14:12:45] [Render thread/ERROR] (FabricLoader) Minecraft has crashed!
 net.fabricmc.loader.impl.FormattedException: java.lang.RuntimeException: test
        at net.fabricmc.loader.impl.FormattedException.ofLocalized(FormattedException.java:63) ~[fabric-loader-0.16.9.jar:?]
        at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:482) ~[fabric-loader-0.16.9.jar:?]
        at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) [fabric-loader-0.16.9.jar:?]
        at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.16.9.jar:?]
        at net.fabricmc.devlaunchinjector.Main.main(Main.java:86) [dev-launch-injector-0.2.1+build.8.jar:?]
Caused by: java.lang.RuntimeException: test
        at knot/net.fabricmc.fabric.test.client.gametest.ClientGameTestTest.lambda$runTest$0(ClientGameTestTest.java:85) ~[testmodClient/:?]
        at knot/net.fabricmc.fabric.impl.client.gametest.ThreadingImpl.runTaskOnThisThread(ThreadingImpl.java:203) ~[fabric-client-gametest-api-v1-1.0.0+db066d5704-dev.jar:?]
        at Async Stack Trace..(Unknown Source) ~[?:?]
        at knot/net.fabricmc.fabric.impl.client.gametest.ThreadingImpl.runTaskOnOtherThread(ThreadingImpl.java:195) ~[fabric-client-gametest-api-v1-1.0.0+db066d5704-dev.jar:?]
        at knot/net.fabricmc.fabric.impl.client.gametest.ThreadingImpl.runOnServer(ThreadingImpl.java:179) ~[fabric-client-gametest-api-v1-1.0.0+db066d5704-dev.jar:?]
        at knot/net.fabricmc.fabric.test.client.gametest.ClientGameTestTest.runTest(ClientGameTestTest.java:84) ~[testmodClient/:?]
        at knot/net.fabricmc.fabric.impl.client.gametest.FabricClientGameTestRunner.lambda$start$0(FabricClientGameTestRunner.java:43) ~[fabric-client-gametest-api-v1-1.0.0+db066d5704-dev.jar:?]
        at knot/net.fabricmc.fabric.impl.client.gametest.ThreadingImpl.lambda$runTestThread$1(ThreadingImpl.java:129) ~[fabric-client-gametest-api-v1-1.0.0+db066d5704-dev.jar:?]
        at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
```

--- a/fabric-client-gametest-api-v1/src/testmodClient/java/net/fabricmc/fabric/test/client/gametest/ClientGameTestTest.java
+++ b/fabric-client-gametest-api-v1/src/testmodClient/java/net/fabricmc/fabric/test/client/gametest/ClientGameTestTest.java
@@ -143,7 +143,6 @@ public class ClientGameTestTest implements FabricClientGameTest {
 
 		{
 			context.waitForScreen(TitleScreen.class);
-			context.clickScreenButton("menu.quit");
 		}
 	}
 

